### ISSUE TITLE
plotHeatmap several --yMax 

### ISSUE THREAD

**Issue by Faitero** [fidelram/deepTools#622]
Thanks for this wonderful tool. I cannot specify different maximum y height on plotheatmap for each bam file:

```
plotHeatmap -m matrix.tab.gz -out genes.pdf --colorMap YlGnBu --zMin -10 -10 -10 -10 -10 -10 -10 -10 -10 --zMax 30 30 30 80 80 80 80 40 40 --startLabel TSS --endLabel TES \
--missingDataColor "#FFF6EB" \
--heatmapHeight 25 \
--yMax 100 100 50 100 100 50 100 100 50
```

```
usage: plotHeatmap [--matrixFile MATRIXFILE] --outFileName OUTFILENAME
                   [--outFileSortedRegions FILE] [--outFileNameMatrix FILE]
                   [--dpi DPI] [--kmeans KMEANS] [--hclust HCLUST] [--help]
                   [--version] [--sortRegions {descend,ascend,no}]
                   [--sortUsing {mean,median,max,min,sum,region_length}]
                   [--sortUsingSamples SORTUSINGSAMPLES [SORTUSINGSAMPLES ...]]
                   [--averageTypeSummaryPlot {mean,median,min,max,std,sum}]
                   [--missingDataColor MISSINGDATACOLOR]
                   [--colorMap COLORMAP [COLORMAP ...]] [--alpha ALPHA]
                   [--colorList COLORLIST [COLORLIST ...]]
                   [--colorNumber COLORNUMBER] [--zMin ZMIN [ZMIN ...]]
                   [--zMax ZMAX [ZMAX ...]] [--heatmapHeight HEATMAPHEIGHT]
                   [--heatmapWidth HEATMAPWIDTH]
                   [--whatToShow {plot, heatmap and colorbar,plot and heatmap,heatmap only,heatmap and colorbar}]
                   [--boxAroundHeatmaps BOXAROUNDHEATMAPS]
                   [--xAxisLabel XAXISLABEL] [--startLabel STARTLABEL]
                   [--endLabel ENDLABEL] [--refPointLabel REFPOINTLABEL]
                   [--regionsLabel REGIONSLABEL [REGIONSLABEL ...]]
                   [--samplesLabel SAMPLESLABEL [SAMPLESLABEL ...]]
                   [--plotTitle PLOTTITLE] [--yAxisLabel YAXISLABEL]
                   [--yMin YMIN] [--yMax YMAX]
                   [--legendLocation {best,upper-right,upper-left,upper-center,lower-left,lower-right,lower-center,center,center-left,center-right,none}]
                   [--perGroup] [--plotFileFormat] [--verbose]
plotHeatmap: error: unrecognized arguments: 20 100
```

Thanks in advance

**Comment by dpryan79** [comment]
I'm not sure where it's seeing `20` in there. Can you try this again and remove the various options until you figure out exactly which one is triggering this? Presumably this is some weird argparse bug.

**Comment by Faitero** [comment]
Its the --yMAx option that triggers this error. It didn't allow including different yMax for the different columns:
```
plotHeatmap -m matrix_Histones.tab.gz -out Histones.pdf --yMax 20 40
```
usage: plotHeatmap [--matrixFile MATRIXFILE] --outFileName OUTFILENAME
                   [--outFileSortedRegions FILE] [--outFileNameMatrix FILE]
                   [--dpi DPI] [--kmeans KMEANS] [--hclust HCLUST] [--help]
                   [--version] [--sortRegions {descend,ascend,no}]
                   [--sortUsing {mean,median,max,min,sum,region_length}]
                   [--sortUsingSamples SORTUSINGSAMPLES [SORTUSINGSAMPLES ...]]
                   [--averageTypeSummaryPlot {mean,median,min,max,std,sum}]
                   [--missingDataColor MISSINGDATACOLOR]
                   [--colorMap COLORMAP [COLORMAP ...]] [--alpha ALPHA]
                   [--colorList COLORLIST [COLORLIST ...]]
                   [--colorNumber COLORNUMBER] [--zMin ZMIN [ZMIN ...]]
                   [--zMax ZMAX [ZMAX ...]] [--heatmapHeight HEATMAPHEIGHT]
                   [--heatmapWidth HEATMAPWIDTH]
                   [--whatToShow {plot, heatmap and colorbar,plot and heatmap,heatmap only,heatmap and colorbar}]
                   [--boxAroundHeatmaps BOXAROUNDHEATMAPS]
                   [--xAxisLabel XAXISLABEL] [--startLabel STARTLABEL]
                   [--endLabel ENDLABEL] [--refPointLabel REFPOINTLABEL]
                   [--regionsLabel REGIONSLABEL [REGIONSLABEL ...]]
                   [--samplesLabel SAMPLESLABEL [SAMPLESLABEL ...]]
                   [--plotTitle PLOTTITLE] [--yAxisLabel YAXISLABEL]
                   [--yMin YMIN] [--yMax YMAX]
                   [--legendLocation {best,upper-right,upper-left,upper-center,lower-left,lower-right,lower-center,center,center-left,center-right,none}]
                   [--perGroup] [--plotFileFormat] [--verbose]
plotHeatmap: error: unrecognized arguments: 40
```

**Comment by dpryan79** [comment]
What version are you using? Support for multiple yMin and yMax values was added in version 2.5.0.

**Comment by Faitero** [comment]
I was using plotHeatmap 2.4.1-4-0eeb2e1 and the issue solved with the latest plotHeatmap 2.5.3. I was updated but still using an outdated binary earlier in the path.

Thanks and sorry for bothering you

**Comment by dpryan79** [comment]
No worries, glad you got it figured out.
